README.md

```diff
@@ -1,30 +1,31 @@
 # Registerables
 
-![Current Version 0.7.2](https://img.shields.io/badge/Current_Version-0.7.2-yellow.svg?style=flat " ") 
+![Current Version 0.8.0](https://img.shields.io/badge/Current_Version-0.8.0-yellow.svg?style=flat " ") 
 [![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/)
 [![GitHub_CI](https://github.com/Pink-Crab/Perique-Registerables/actions/workflows/php.yaml/badge.svg)](https://github.com/Pink-Crab/Perique-Registerables/actions/workflows/php.yaml)
 [![codecov](https://codecov.io/gh/Pink-Crab/Perique-Registerables/branch/master/graph/badge.svg?token=R3SB4WDL8Z)](https://codecov.io/gh/Pink-Crab/Perique-Registerables)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/Pink-Crab/Perique-Registerables/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/Pink-Crab/Perique-Registerables/?branch=master)
 
 A collection of Abstract Classes for creating common WordPress fixtures which need registering.
 
-* Post Types
-* Taxonomies
-* Meta boxes
-* Meta Data
+- [Post Type](#post-type)  
+- [Taxonomy](#taxonomy)  
+- [Meta Box](#meta-box)
+  - [Shared Meta Boxes](#shared-meta-boxes)
+- [Meta Data](#meta-data)
+  - [Additional Meta Data](#additional_meta_data_controller)
+
 
 
 ## Version ##
 
-**Release 0.7.2**
+**Release 0.8.0**
 
 > For older versions of the PinkCrab Plugin Framework please use Registerables V0.5.\*
 
 ## Why? ##
 
-Creating many of WordPress's internal fixtures can sometimes be very verbose with large arrays of values which do not throw errors if incorrect. 
-
-The PinkCrab Registerables module provides a small selection of Abstract Classes which can be extended and added to the registration system.  
+WordPress has a number of Registerable functions based around Post Types, Post Meta and Taxonomies. These tend to have large arrays of args that need to be defined. This small library allows these to be defined as Classes which can be registered via the Registration process.
```

Comment:
Maybe something like this:

WordPress has a number of Registerable functions for Post Types, Post Meta and Taxonomies. These tend to require large arrays of arguments to be defined. This library provides Classes which can be registered and used with the Registration process.

Reply:
changed, thanks